### PR TITLE
embed

### PR DESCRIPTION
**Pull request by itsokk**

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -238,7 +238,12 @@ client.on('messageUpdate', (oldMessage, newMessage) => {
 client.on("messageDelete", (msg) => {
     if(msg.content.startsWith(config.prefix)) {
         if (!msg.member.hasPermission("CHANGE_NICKNAME")) {
-            msg.channel.send(`Message ${msg.content} by ${msg.author.tag} was deleted.`)
+            const deleteEmbed = new Discord.MessageEmbed()
+                .setAuthor("カミブルー！", "https://cdn.discordapp.com/avatars/638403216278683661/1e8bed04cb18e1cb1239e208a01893a1.png", "https://kamiblue.org")
+                .setDescription(`Message ${msg.content} by ${msg.author.tag} was deleted.`)
```
Comment (dominikaaaa) [inline main.js:243]: this should have `\`` around the message content 

Also it should ping the users, so do `<@${msg.author.id}>`

Comment (itsokk) [reply main.js:243]: ok sure, sorry

Comment (itsokk) [reply main.js]: works?

Comment (dominikaaaa) [reply main.js]: put ` around message content

Comment (itsokk) [reply main.js]: what why it worked for me

Comment (dominikaaaa) [reply main.js]: for formatting?

Comment (itsokk) [reply main.js]: should be done

**Pull request merged**